### ISSUE TITLE
How to clear cookies (iOS)

### ISSUE THREAD

**Issue by scitbiz** [fluttercommunity/flutter_webview_plugin#200]
The options `clearCookies` work perfectly on Android but not working on iOS
```
webview.launch(
    "https://google.com",
    clearCache: true,
    clearCookies: true,
);
```

**Comment by YukiOya** [comment]
@scitbiz 
What Xcode version do you use?
`WKWebsiteDataTypeFetchCache` is implemented from iOS 11.3.
If you use older, you try to update Xcode 10 or later. It maybe resolve.

**Comment by scitbiz** [comment]
> @scitbiz
> What Xcode version do you use?
> `WKWebsiteDataTypeFetchCache` is implemented from iOS 11.3.
> If you use older, you try to update Xcode 10 or later. It maybe resolves.

I'm using Xcode 9.2 but I think it's not a problem anyway.
The problem may come from flutter v0.10.2. I've downgraded to v0.10.1 and it works fine

**Comment by slightfoot** [comment]
Should be fixed as of v0.3.0+2. Please retest.

**Comment by scitbiz** [comment]
> Should be fixed as of v0.3.0+2. Please retest.

Hi,
After update to v0.3.0+2, my webview no longer show up in the screen.

**Comment by bagintz** [comment]
For whatever it is worth, clearing cookies did not work for me either on v0.3.0+2

**Comment by colelawrence** [comment]
The PR works for me, the current version does not unfortunately.

**Comment by CaleyD** [comment]
This PR works for our application as well 👍

**Comment by edwardez** [comment]
As of 0.3.5, `clearCookies` is still not working for me on iOS 12.2 simulator, however the pr works like a charm!

**Comment by nnecec** [comment]
please solve this problem

**Comment by chynnasevilleno** [comment]
Any updates on this? Been having the same problem.

**Comment by charafau** [comment]
fixed in 0.3.7

**Comment by gugadev** [comment]
> fixed in 0.3.7

The problem persists in 0.4.0 for me.
